pkgs/tools/misc/poweralertd/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, meson, ninja, pkg-config, scdoc, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "poweralertd";
+  version = "0.1.0";
+
+  outputs = [ "out" "man" ];
+
+  src = fetchurl {
+    url = "https://git.sr.ht/~kennylevinsen/poweralertd/archive/${version}.tar.gz";
+    sha256 = "11b0lgrp7kyysvc8gh8w37ydig74j2bg0y885ir73w3pym6sy65a";
```

Comment:
Can you switch to fetchFromSourcehut https://github.com/NixOS/nixpkgs/pull/102225?